### DISCUSSION TITLE
The toolbar texture gets bigger, drawing page duplicates on the back.

### DISCUSSION THREAD

**Discussion by DomcTriesToAnimate** [opentoonz/opentoonz#6578] [Q&A]
The toolbar texture would get bigger and the drawing page duplicates on the back. It does that everytime I start drawing. It should've been the same as it was when I opened the software. And if I undo until the frame is blank, everything would go back to normal. I once tried to uninstall and then reinstall it, but the problem is still there. here are the screen shots:

![Screen-05-08-2020_125535](https://user-images.githubusercontent.com/69233070/89377053-3c102000-d71b-11ea-8378-f7f165976358.png)
Before undo

![Screen-05-08-2020_125639](https://user-images.githubusercontent.com/69233070/89377090-4cc09600-d71b-11ea-8c97-1ad00fefedd1.png)
After undo

Note: It's the OS version of OpenToonz

**Comment by manongjohn** [comment]
This has been brought up a number if times.

3 things you can do...

Short term: Undock style editor.

Permanent fix:

If you have an Intel Graphics display driver, you may need to find and download the latest version. Don't rely on Driver Update as it may not detect it. Need to go onto intel's driver download site and check the version of the driver you have against yours. Manually download and install the driver upgrade it if you are not at the latest.

If you have driver version 26.20.100.7463 or a version of this graphic driver, try downloading and installing version 26.20.100.7584. Here is the direct download link:

https://downloadmirror.intel.com/29274/a08/igfx_win10_100.7584.exe


If you have an NVIDIA graphics card, this may work better for you: Enter the NVIDIA Control Panel and create a 3D Profile for OpenToonz.exe and if given the option, set to use High Performance GPU. You are essentially forcing OT to use your GPU which should fix the issue.

Alternatively, if using Windows 10, it might help to search for Graphics Settings in Windows Settings. Add OpenToonz.exe and configure for High Performance.
